endpoint.go

```diff
@@ -360,6 +368,14 @@ func (ep *endpoint) Leave(containerID string, options ...EndpointOption) error {
 		}
 	}
 
+	// Remove non-interface routes.
+	for _, r := range ep.joinInfo.StaticRoutes {
+		err = sb.RemoveStaticRoute(r)
+		if err != nil {
+			logrus.Debugf("Remove route failed: %v", err)
+		}
+	}
```

Comment:
I had a comment which got overwritten. What about interface routes? Would they get removed when we remove interfaces or does the kernel not allow interfaces to be removed when there are active routes referencing the interface?